Makefile

```
@@ -322,7 +322,7 @@ endif
 # This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
-catalog-build: opm bundle-push ## Generate operator-catalog dockerfile using the operator-bundle image built and published above; then build catalog image
+catalog-build: bundle-push opm ## Generate operator-catalog dockerfile using the operator-bundle image built and published above; then build catalog image
```

Why is the order changed? I think we need `opm` target first, as it is invoked below

```suggestion
catalog-build: opm bundle-push ## Generate operator-catalog dockerfile using the operator-bundle image built and published above; then build catalog image
```

---

The bundle image must be pushed first before catalog image cab be built. opm installation is also required. The order does not matter. I switched them as it is more natural to start the catalog build right after opm is installed.

---

Okay - thanks.

---

Yes this is expected, because the host is already set. "subdomain is a DNS subdomain that is requested within the ingress controller's domain (as a subdomain). If host is set this field is ignored."
https://github.com/openshift/api/blob/master/route/v1/types.go#L99